### ISSUE TITLE
SSL disable flag for localhost compiles!

### ISSUE THREAD

**Issue by osevan** [dragonflydb/dragonfly#128]
We need ssl disabling, because its bot needed in localhost in memory environment. 

But at compile time to avoid overhead. 

Thanks and

Best regards

**Comment by osevan** [comment]
option(DF_USE_SSL "Provide support for SSL connections" ON)

Its enough to set here to off?

**Comment by romange** [comment]
yes. when you configure the build with cmake: `cmake .. -DDF_USE_SSL=OFF`
